src/storage/txn/actions/prewrite.rs

```diff
@@ -440,6 +454,11 @@ impl<'a> PrewriteMutation<'a> {
             self.txn_props.txn_size,
             self.min_commit_ts,
         );
+        // Only Lock needs to record `last_change_ts` in its write record, write records
+        // of other types themselves are effective changes.
```

Comment:
Nit: ROLLBACK doesn't, though it doesn't matter at all

Reply:
Ah, I mean the possible LockType here. We don't have a `ROLLBACK` lock.